app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt

```diff
@@ -85,58 +94,90 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
                     when (it) {
                       UserViewModel.SignInFailureReason.ACCOUNT_NOT_FOUND -> {
                         Toast.makeText(context, accountNotFoundToast, Toast.LENGTH_LONG).show()
+                        navigationActions.navigateTo(Screen.CREATE_PROFILE)
                       }
                       UserViewModel.SignInFailureReason.ERROR -> {
                         Toast.makeText(context, failSignInMsg, Toast.LENGTH_LONG).show()
                       }
                     }
                   })
             }
-            // Create Account Button
-            Button(
-                text = "Create an account",
-                colorLevel = ColorLevel.SECONDARY,
-                modifier =
-                    Modifier.fillMaxWidth()
-                        .border(BorderStroke(1.dp, Color.LightGray), RoundedCornerShape(50))
-                        .height(48.dp)
-                        .width(250.dp),
-                onClick = { navigationActions.navigateTo(Screen.CREATE_PROFILE) },
-                testTag = "CreateAccountButton")
-
-            // Anonymous Login Button
-            Button(
-                text = stringResource(R.string.sign_in_guest_button),
+            // Seperate main button from other buttons with a spacer
+            Spacer(modifier = Modifier.height(16.dp))
+            val alphaOfGuestButton = remember { Animatable(1f) }
+            val alphaOfOfflineButton = remember { Animatable(1f) }
+            suspend fun clickAnimation(alpha: Animatable<Float, AnimationVector1D>) {
+              alpha.animateTo(
+                  targetValue = 0.1f,
+                  animationSpec =
+                      tween(
+                          durationMillis = 100,
+                      ))
+            }
+            fun resetAlpha(alpha: Animatable<Float, AnimationVector1D>) {
```

Comment:
If you are using this function only inside this class maybe you can put it as private ?

Reply:
This function is already a local function, It can't be used outside of the SignInScreen. The compiler will not even let me write private

Reply:
Ok it is perfect then !